src/appshell/qml/Preferences/AppearancePreferencesPage.qml

```diff
@@ -49,26 +49,85 @@ PreferencesPage {
 
         readonly property int firstColumnWidth: 212
 
-        ThemesSection {
-            width: parent.width
+        StyledTextLabel {
+            text: highContrastEnable.checked ? qsTrc("appshell", "High Contrast Themes") : qsTrc("appshell", "Themes")
+            font: ui.theme.bodyBoldFont
+        }
 
-            themes: appearanceModel.themes
-            currentThemeIndex: appearanceModel.currentThemeIndex
+        CheckBox {
+            id: highContrastEnable
+            width: 200
+            checked: appearanceModel.enableHighContrastChecked()
 
-            onThemeChangeRequested: {
-                appearanceModel.currentThemeIndex = newThemeIndex
+            text: "Enable High-Contrast"
+            onClicked: {
+                checked = !checked
+                highContrastEnable.checked ? appearanceModel.loadLastUsedHighContrastTheme() : appearanceModel.loadLastUsedGeneralTheme()
             }
         }
 
-        AccentColorsSection {
-            width: parent.width
+        Loader {
+            sourceComponent: highContrastEnable.checked ? highContrastThemesModel : generalThemesModel
+        }
 
-            colors: appearanceModel.accentColors
-            currentColorIndex: appearanceModel.currentAccentColorIndex
-            firstColumnWidth: parent.firstColumnWidth
+        Component {
+            id: generalThemesModel
+
+            Column {
+                spacing: 24
```

Comment:
Better to create a private readonly property and reuse it:

QObject {
id: prv

readonly property int spacing: 24
}

...

Column {
spacing: prv.spacing
}

Reply:
Understood!
Should i also make this change for `firstColumnWidth` declared in the same file? i.e. declare it inside the QtObject and refer to it as `prv.firstColumnWidth`?

Reply:
"Should i also make this change for firstColumnWidth declared in the same file?"

Yes, you can move this property to prv also

Reply:
Fixed :)